components/book/Update.tsx

```diff
@@ -9,12 +8,14 @@ interface Props {
 }
 
 export const Update: FunctionComponent<Props> = ({ book }) => {
+	const [error, setError] = useState(null);
+
 	const handleDelete = () => {
 		if (window.confirm("Are you sure you want to delete this item ?")) {
 			try {
 				fetch(`${book["@id"]}`, { method: "delete" });
 			} catch (error) {
-				alert("Error when delete element");
+				setError(error);
```

Comment:
```suggestion
				setError("Error when deleting the resource.");
```

IMHO it's not useful to display the technical error to the end user.